spec/envied_spec.rb

```diff
@@ -215,6 +214,7 @@ def envied_require(*args)
     end
 
     describe "::required?" do
+      # TODO: change to always return boolean
       it 'yields true-ish when ::require is called' do
```

Comment:
@eval what do you think of changing `ENVied.required?` to always return a boolean instead? My preference is for explicit values and not returning the `@env` for that method.